Core/Inc/FreeRTOSConfig.h

```diff
@@ -75,7 +75,9 @@
 #define configUSE_16_BIT_TICKS                   0
 #define configUSE_MUTEXES                        1
 #define configQUEUE_REGISTRY_SIZE                8
+#define configCHECK_FOR_STACK_OVERFLOW           1

```

Comment:
https://www.freertos.org/Documentation/02-Kernel/02-Kernel-features/09-Memory-management/02-Stack-usage-and-stack-overflow-checking.  I think we want method 2.

Also do we want our own hook?
``` 
void vApplicationStackOverflowHook( TaskHandle_t xTask,
                                    char *pcTaskName );
```
If we experience perf issues we may need to disable this, but we prob wont with only three threads.